dts/bindings/ethernet/microchip,enc28j60.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2018, Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: 10Base-T Ethernet Controller with SPI Interface
+version: 0.1
+
+description: >
+    This binding gives a base representation of the standalone ENC28J60 chip
+inherits:
+  !include ethernet.yaml
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "microchip,enc28j60"
+    spi-port-name:
+      type: string
+      description: name for the port being used by the chip
+      generation: define
+      category: required
+    spi-frequency:
+      type: integer
+      description: host's spi frequency
+      generation: define
+      category: required
+    int-gpios:
+      type: compound
+      category: required
+      generation: define, use-prop-name
+    spi-gpios:
```

Comment:
Actually that information should come from the SPI master node (it's counter intuitive here in Zephyr, but since we have to keep DT compatibility with Linux and since Linux does it that way: here is the reason.). Check some other slaves that are used in spi master node providing cs-gpios attribute.